### ISSUE TITLE
chech_ping claims packet loss is 33% with 2 packets sent

### ISSUE THREAD

**Issue by paddg** [nagios-plugins/nagios-plugins#774]
```
# /usr/lib/nagios/plugins/check_ping --version
check_ping v2.3.5 (monitoring-plugins 2.3.5)
```
Example:
```
/usr/lib/nagios/plugins/check_ping -4 -H 10.10.10.3 -p 2 -w 3000,80% -c 5000,100%
PING OK - Packet loss = 33%, RTA = 24.64 ms|rta=24.643000ms;3000.000000;5000.000000;0.000000 pl=33%;80;100;0;
```
If the check fails due to a network problem, we always see **33% packet loss**. But mathematically, that cannot be the case with 2 packets sent. And we have checked it, with `-p 2` only 2 packets are actually sent.
